README.md

```diff
@@ -1,20 +1,22 @@
-
 # groundhog
+`groundhog` is a service and client libraries to access NASA's publicly available surface elevation data, [SRTM](https://www2.jpl.nasa.gov/srtm/).  Given a table with GPS information, this package provides utilities to enrich that dataset with elevation and slope features.  Just load the container and one of the clients, and you're all set! 
 
-Our mission:
-
-> Things go up. And down. We might be able to make money by telling people when that happened.
+***
+## Getting Started
 
-Our motto:
+Clone this repository. 
 
-> If you ain't first, you're last.
+```bash
+git clone https://github.com/uptake/groundhog.git
+cd groundhog
+```
 
 ### Building the container
 
-We'll put the container up on Dockerhub soon. For now, you can build locally from this repo.
+We'll put the container up on Dockerhub soon ([status](https://github.com/uptake/groundhog/issues/24)). For now, you can build locally from this repo.
 
 ```bash
-docker build -t groundhog -f Dockerfile .
+docker build -t groundhog -f Dockerfile-app .
```

Comment:
we have #5 so this will get reverted, but I like the idea of leaving that alone for someone who is less familiar with open source to pick up as a "good first issue".

So I support the change in this PR.